src/definition/populate_test.go

```diff
@@ -456,75 +421,97 @@ func TestIntegrationProtocol2PopulateFunc_PopulateOnlySpecifiedGroups(t *testing
 		},
 	}
 
-	expectedEntityData1, err := sdk.NewEntityData("testEntity11-generated", "playground:test")
-	if err != nil {
-		t.Fatal()
-	}
+	// Create a dummy integration, used only to create entities easily
+	dummyIntgr, err := integration.New("nr.test", "1.0.0")
```

Comment:
To indicate that this is integration under test?
```suggestion
	testIntgr, err := integration.New("nr.test", "1.0.0")
```

Reply:
This was a leftover from a refactor (which I forgot), I think it should be fine to name it just `intgr` like the rest